src/meta_utils.py

```diff
@@ -0,0 +1,528 @@
+import numpy as np
+import pandas as pd
+from typing import Optional, Tuple, Callable
+
+# (1) Crop land **mapping** <- MOST GENERAL
+#     -> NOTE: With crop land map there is no 'final' agreement between two labeler 
+#              sets b/c there is typically no *forced* agreement or resolvement.
```

Comment:
I just want to make a note that this could be a path towards forcing resolution. It is ignored simply for time saving.